LoadTests.cmake

```diff
@@ -0,0 +1,47 @@
+# This file is include()'d by CTest. It executes test-libmongoc to get a list
+# of all tests that are registered. Each test is then defined as a CTest test,
+# allowing CTest to control the execution, parallelization, and collection of
+# test results.
```

Comment:
This file feels out of place being in project root directory. Should it be moved into `build/cmake`?

Reply:
Probably a better place, yes.